Changes for Powerflex Manual Pipeline

nikitajoshi1 opened the pull request:
# Description
Updated the lint library , fixed lint errors and security vulnerabilities.
Pipeline Build: https://osj-sio-03-prd.cec.lab.emc.com/job/Terraform-Pipelines/job/tf-powerflex-manual/40/

# GitHub Issues


# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 80% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

doriac11 commented on powerflex/provider/device_resource_test.go:
```diff
@@ -45,14 +45,14 @@ var createStoragePool = `
-		use_rmcache = true
-	}
-`
+// var updateStoragePool = `
```
should this be removed?

nikitajoshi1 replied:
@doriac11 I didn't delete things as I didn't want to remove all the existing resource variables in case team wants to implement related test cases in near future. So just commented for now to resolve lint issues. Will remove if the team thinks they are not going to use it.

AnikaAgiwal2711 replied:
What exact lint issue are you getting if you uncomment this ??

nikitajoshi1 replied:
@AnikaAgiwal2711 `unusedVariable` errors

nikitajoshi1 commented on powerflex/provider/sds_resource_test.go:
```diff
@@ -1503,106 +1503,108 @@ func TestSDSResourceRename(t *testing.T) {
-    ]
-}
-`
+// var addSDSInvalidRole = `
```
@doriac11 Same reason as above.

rounak-adhikary commented:
I dont think these comments were serving any purpose. Its okay to remove.
Everything else lgtm.

nikitajoshi1 commented:
> > I dont think these comments were serving any purpose. Its okay to remove. The rest lgtm.
> 
> Thanks for the update @rounak-adhikary . Will remove those.



> > I dont think these comments were serving any purpose. Its okay to remove. The rest lgtm.
> 
> Thanks for the update @rounak-adhikary . Removed those and updated PR. Please approve if changes look okay.

gokul-srivathsan reviewed (approved):
LGTM
